README.md

```diff
@@ -205,9 +205,9 @@ attaches to the one initialized by the app. An example of the engine initializat
 
 On MacOS, Diligent Engine supports OpenGL and Vulkan backends. Initialization of GL context on MacOS is
 performed by the application, and the engine attaches to the context created by the app; see
-[GLView.m](https://github.com/DiligentGraphics/DiligentEngine/blob/master/Common/NativeApp/Apple/Source/Classes/OSX/GLView.m)
+[GLView.m](https://github.com/DiligentGraphics/DiligentTools/tree/master/NativeApp/Apple/Source/Classes/OSX/GLView.m)
```

Comment:
Thanks, did not know these were broken. Could you submit this a separate PR?

Reply:
Yes, I can submit these fix in another PR. I believe there were other broken links too. I will try to find and fix them all.